sekoia_automation/scripts/sync_library.py

```diff
@@ -452,9 +463,11 @@ def execute(self):
                 module_path=self.modules_path.joinpath(self.module).absolute(),
             )
 
-    def _get_module_docker_name(self, manifest: dict) -> str:
+    def _get_module_docker_info(self, manifest: dict) -> DockerInfo:
         if docker := manifest.get("docker"):
-            return docker
+            (name, version, *extra) = docker.split(":")
+            return DockerInfo(name, version)
```

Comment:
Do we have manifests with the `docker` attribute ?

Maybe we should stop supporting it since it may lead to a lot of issues:
- Registry not being set
- version in the attribute not the same as the "real" module version

Reply:
I took in account your comment with [this PR](https://github.com/SEKOIA-IO/sekoia-automation-sdk/pull/149)